src/main/java/edu/kis/powp/jobs2d/drivers/command/OperateToCommand.java

```diff
@@ -4,21 +4,15 @@
 
 public class OperateToCommand implements DriverCommand {
 
-    int x;
-    int y;
-
-    Job2dDriver driver = new Job2dDriver() {
-        @Override
-        public void setPosition(int i, int i1) {
-
-        }
-
-        @Override
-        public void operateTo(int i, int i1) {
-
-        }
-    };
-
+    private final int x;
+    private final int y;
+    private final Job2dDriver driver;
+
+    public OperateToCommand(Job2dDriver driver, int x, int y) {
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).